src/welcome/helpers/phrase.ts

```diff
@@ -0,0 +1,36 @@
+import Mnemonic from 'bitcore-mnemonic'
+
+export const ERROR_MESSAGES = {
+  invalidLength: 'Invalid seed length',
+  invalidPhrase: 'Invalid phrase',
+  invalidWords: (words: string[]) =>
+    `The following words are invalid: "${words.join(' ')}"`,
+}
+
+export function getDictionary(): string[] {
+  return Mnemonic.Words.ENGLISH
+}
+
+function isWordInDictionary(word: string): boolean {
+  const dictionary = getDictionary()
+  return dictionary.includes(word)
+}
+
+export function generatePhrase(): string {
+  return new Mnemonic(getDictionary()).toString()
```

Comment:
By default it uses ENGLISH, so this is not changing the previous behaviour, it's just making it explicit.